yarn-project/archiver/src/archiver/memory_archiver_store/memory_archiver_store.test.ts

```diff
@@ -23,7 +23,13 @@ describe('MemoryArchiverStore', () => {
 
       await archiverStore.addBlocks(blocks);
       await Promise.all(
-        blocks.map(block => archiverStore.addLogs(block.newEncryptedLogs, block.newUnencryptedLogs, block.number)),
+        blocks.map(block =>
+          archiverStore.addLogs(
+            new L2BlockL2Logs(blocks[0].body.txEffects.map(txEffect => txEffect.logs!.encryptedLogs)),
```

Comment:
We have this conversion of txEffects to L2BlockL2Logs all over the place now. Could you implement the following 2 getters on Body and use it in all these places?

get encryptedLogs(): L2BlockL2Logs;
get unencryptedLogs(): L2BlockL2Logs;

I think it will make the code much cleaner.